test.sh

```diff
@@ -86,6 +86,9 @@ function runTests4() {
   export CONJUR_FOLLOWER_URL="https://conjur_4-follower/api"
   export CONJUR_SSL_CERTIFICATE="$(cat tmp/conjur.pem)"
 
+  # don't run the integration tests for v4
```

Comment:
Thanks, I actually had realized this same thing yesterday and fixed it as part of #76: 
https://github.com/cyberark/conjur-service-broker/pull/76/commits/217c1f23bf7fa8bce27ad771cc051cf094c74d4d#diff-a0a6573232f8a72ee6a10c68f0b1f5bdR1

I think I prefer the cuke tag method for the selective test run (and I also have a comment there describing why). What do you think?

Reply:
that makes sense to me :) I'll drop my commit